RocketChat/Rocket.Chat
# Regression: invalid statistics format

## Proposed changes (including videos or screenshots)





## Issue(s)

This property was breaking statistics collection, it expected ISO strings for dates.

## Steps to test or reproduce

Clone the pr, start the workspace, notice the lack of messages about statistics being broken.

## Further comments
